app/routes/overview/components/LatestReadme.css

```diff
@@ -25,3 +24,11 @@
     transform: scale(1.05);
   }
 }
+
+.expand {
+  transition: transform var(--easing-medium);
+}
+
+.rotate {
+  transform: rotate(90deg);
+}
```

Comment:
In my head, 270 degreees would make more sense (going from ➡️ to ⬆️) or even starting at ⬇️ and changing it to 270.

Reply:
In my head, 90 degrees makes more sense when you're actually moving 90 degrees

Reply:
ofc, but having the icon to collapse the section look like this ⬇️ seems off

Reply:
No it looks like ➡️, and then after it is triggered (and rotated) it looks like ⬇️

Reply:
Yes, the point was having it point down to collapse it (upwards) seems off.

Reply:
I'm not sure if I'm following you. You mean it seems off that the arrow is pointing down when the content is expanded? Does it not make sense for the arrow to "follow"? I feel like that's the norm

Reply:
So i found a few faq pages which have these kind of icons. [Instagram](https://help.instagram.com/280908123309761), [Helsenorge](https://www.helsenorge.no/koronavirus) and [SAS](https://www.sas.no/hjelp-og-kontakt/ofte-stilte-sporsmal/?faq-category=baggage&faq-search=) all behave like I would expect in my mind - the SAS one animates exactly like i meant. I did also find a [Microsoft FAQ](https://learn.microsoft.com/en-us/answers/support/qna-faq) where the sidebar has your style of button, so I guess there is a precedent for that too. Also, this is not worth spending more time on and i think your approach looks just fine, so :shipit:

Reply:
The three examples you refer to all rotate 180 degrees. This one only rotates 90 degrees, like your last example, hence their resembles. Both are, in my opinion, valid designs, and there seems to be [no right way](https://ux.stackexchange.com/questions/5735/which-way-should-arrows-point-in-a-collapsible-accordion-left-down-down-up).